packages/firebase_analytics/firebase_analytics/example/lib/main.dart

```diff
@@ -72,7 +72,7 @@ class _MyHomePageState extends State<MyHomePage> {
         'int': 42,
         'long': 12345678910,
         'double': 42.0,
-        'bool': true,
+        'bool': true.toString(), // Only strings and integers are supported for GA custom event parameters https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview
```

Comment:
Well done on figuring out the branch change 😄. Do you mind changing the wording and formatting to below noted?  
```dart
		'int': 42,
        'long': 12345678910,
        'double': 42.0,
		// Only strings and numbers (ints & doubles) are supported for GA custom event parameters:
		// https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets#overview
        'bool': true,
```